CorsixTH/Lua/game_ui.lua

```diff
@@ -554,11 +554,19 @@ function GameUI:onMouseMove(x, y, dx, dy)
 
   if self:_isMouseScrollButtonDown() then
     local zoom = self.zoom_factor
-    self.current_momentum.x = -dx/zoom
-    self.current_momentum.y = -dy/zoom
+    self.current_momentum.x = self.current_momentum.x - dx/zoom
```

Comment:
Indentation is off

Reply:
I'm sorry, i don't know what is the problem here, the code is indented using 2 spaces per indent level following the coding conventions of the wiki, in this line there is 4 spaces because the code is indented 2 times. Can you elaborate on this more precisely?

Reply:
Cyco I can't see that issue either. The indent is 4 spaces and not tabs which is right. Is there a hidden character or something?

Reply:
```bash
U+0020 : SPACE [SP]
U+0020 : SPACE [SP]
U+0020 : SPACE [SP]
U+0020 : SPACE [SP]
U+0073 : LATIN SMALL LETTER S
U+0065 : LATIN SMALL LETTER E
U+006C : LATIN SMALL LETTER L
U+0066 : LATIN SMALL LETTER F
U+002E : FULL STOP {period, dot, decimal point}
U+0063 : LATIN SMALL LETTER C
U+0075 : LATIN SMALL LETTER U
U+0072 : LATIN SMALL LETTER R
U+0072 : LATIN SMALL LETTER R
U+0065 : LATIN SMALL LETTER E
U+006E : LATIN SMALL LETTER N
U+0074 : LATIN SMALL LETTER T
U+005F : LOW LINE
U+006D : LATIN SMALL LETTER M
U+006F : LATIN SMALL LETTER O
U+006D : LATIN SMALL LETTER M
U+0065 : LATIN SMALL LETTER E
U+006E : LATIN SMALL LETTER N
U+0074 : LATIN SMALL LETTER T
U+0075 : LATIN SMALL LETTER U
U+006D : LATIN SMALL LETTER M
U+002E : FULL STOP {period, dot, decimal point}
U+0078 : LATIN SMALL LETTER X
U+0020 : SPACE [SP]
U+003D : EQUALS SIGN
U+0020 : SPACE [SP]
U+0073 : LATIN SMALL LETTER S
U+0065 : LATIN SMALL LETTER E
U+006C : LATIN SMALL LETTER L
U+0066 : LATIN SMALL LETTER F
U+002E : FULL STOP {period, dot, decimal point}
U+0063 : LATIN SMALL LETTER C
U+0075 : LATIN SMALL LETTER U
U+0072 : LATIN SMALL LETTER R
U+0072 : LATIN SMALL LETTER R
U+0065 : LATIN SMALL LETTER E
U+006E : LATIN SMALL LETTER N
U+0074 : LATIN SMALL LETTER T
U+005F : LOW LINE
U+006D : LATIN SMALL LETTER M
U+006F : LATIN SMALL LETTER O
U+006D : LATIN SMALL LETTER M
U+0065 : LATIN SMALL LETTER E
U+006E : LATIN SMALL LETTER N
U+0074 : LATIN SMALL LETTER T
U+0075 : LATIN SMALL LETTER U
U+006D : LATIN SMALL LETTER M
U+002E : FULL STOP {period, dot, decimal point}
U+0078 : LATIN SMALL LETTER X
U+0020 : SPACE [SP]
U+002D : HYPHEN-MINUS {hyphen, dash; minus sign}
U+0020 : SPACE [SP]
U+0064 : LATIN SMALL LETTER D
U+0078 : LATIN SMALL LETTER X
U+002F : SOLIDUS {slash, forward slash, virgule}
U+007A : LATIN SMALL LETTER Z
U+006F : LATIN SMALL LETTER O
U+006F : LATIN SMALL LETTER O
U+006D : LATIN SMALL LETTER M
```
Nothing here hidden, indent is correct for this part of the function, will mark this resolved.